packages/vaadin-virtualizer/src/iron-list.js

```diff
@@ -267,56 +282,51 @@ export const PolymerIronList = Class({
    * @type {number}
    */
   get lastVisibleIndex() {
-    let idx = this._lastVisibleIndexVal;
+    var idx = this._lastVisibleIndexVal;
     if (idx == null) {
-      let physicalOffset = this._physicalTop + this._scrollOffset;
-      this._iterateItems(function (pidx, vidx) {
-        if (physicalOffset < this._scrollBottom) {
-          idx = vidx;
-        }
-        physicalOffset += this._physicalSizes[pidx];
-      });
+      if (this.grid) {
```

Comment:
The changes related to the `grid` mode can be reverted if we don't want to support it.